config/tokens.json

```diff
@@ -302,261 +337,6 @@
-      },
-      "2xl": {
-        "value": {
-          "fontFamily": "$fontFamilies.fira-sans",
-          "fontWeight": "{fontWeights.regular}",
-          "lineHeight": "{lineHeights.sm}",
-          "fontSize": "{fontSize.2xl}",
-          "letterSpacing": "$letterSpacing.0",
-          "paragraphSpacing": "$paragraphSpacing.0",
-          "textCase": "{textCase.none}",
-          "textDecoration": "$textDecoration.none"
-        },
-        "type": "typography",
-        "description": "2XL font size is used for secondary headlines on desktop and primary headlines on mobile."
-      },
-      "2xl-bold": {
-        "value": {
-          "fontFamily": "$fontFamilies.fira-sans",
-          "fontWeight": "{fontWeights.bold}",
-          "lineHeight": "{lineHeights.sm}",
-          "fontSize": "{fontSize.2xl}",
-          "letterSpacing": "$letterSpacing.0",
-          "paragraphSpacing": "$paragraphSpacing.0",
-          "textCase": "{textCase.none}",
-          "textDecoration": "$textDecoration.none"
-        },
-        "type": "typography",
-        "description": "2XL font size is used for secondary headlines on desktop and primary headlines on mobile."
-      },
-      "xl": {
-        "value": {
-          "fontFamily": "$fontFamilies.fira-sans",
-          "fontWeight": "{fontWeights.regular}",
-          "lineHeight": "{lineHeights.md}",
-          "fontSize": "{fontSize.xl}",
-          "letterSpacing": "$letterSpacing.0",
-          "paragraphSpacing": "$paragraphSpacing.0",
-          "textCase": "{textCase.none}",
-          "textDecoration": "$textDecoration.none"
-        },
-        "type": "typography",
-        "description": "XL font size is used for large sub-headlines and secondary headlines on mobile."
-      },
-      "xl-bold": {
-        "value": {
-          "fontFamily": "$fontFamilies.fira-sans",
-          "fontWeight": "{fontWeights.bold}",
-          "lineHeight": "{lineHeights.md}",
-          "fontSize": "{fontSize.xl}",
-          "letterSpacing": "$letterSpacing.0",
-          "paragraphSpacing": "$paragraphSpacing.0",
-          "textCase": "{textCase.none}",
-          "textDecoration": "$textDecoration.none"
-        },
-        "type": "typography",
-        "description": "XL font size is used for large sub-headlines and secondary headlines on mobile."
-      },
-      "large": {
-        "value": {
-          "fontFamily": "$fontFamilies.fira-sans",
-          "fontWeight": "{fontWeights.regular}",
-          "lineHeight": "{lineHeights.lg}",
-          "fontSize": "{fontSize.lg}",
-          "letterSpacing": "$letterSpacing.0",
-          "paragraphSpacing": "$paragraphSpacing.md",
-          "textCase": "{textCase.none}",
-          "textDecoration": "$textDecoration.none"
-        },
-        "type": "typography",
-        "description": "Large font size is used for small sub-headings, button- and input-labels."
-      },
-      "large-bold": {
```

Comment:
`large-bold` is still referenced by `org.typo.button-text.value`, that's why we get this error:

Reference doesn't exist: org.typo.button-text.value tries to reference text.large-bold